app/parse.py

```diff
@@ -8,8 +15,55 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_one_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").text
```

Comment:
In parse_one_quote, you use .text to get the content of the tags and author, but .get_text(strip=True) might be more efficient and clean, as it removes any leading and trailing whitespace.